pygmt/helpers/decorators.py

```diff
@@ -14,30 +14,30 @@
 
 COMMON_OPTIONS = {
     "R": """\
-        R : str or list
+        region : str or list
             *Required if this is the first plot command*.
             ``'xmin/xmax/ymin/ymax[+r][+uunit]'``.
             Specify the region of interest.""",
     "J": """\
-        J : str
+        projection : str
             *Required if this is the first plot command*.
             Select map projection.""",
     "B": """\
-        B : str or list
+        frame : str or list
             Set map boundary frame and axes attributes.""",
     "U": """\
-        U : bool or str
+        logo : bool or str
```

Comment:
**-U** option is called **timestamp** in GMT, and **stamp** in GMT.jl. See https://github.com/GenericMappingTools/gmt/blob/362acc1302701ff3fe169638de5346f317e44670/src/gmt_init.c#L370. I think we should follow the GMT long-name options. 
```suggestion
        timestamp : bool or str
```

Reply:
Cool, good to have a standard to follow. I'll batch commit these changes.